internal/backend/runtime/omni/controllers/omni/cluster_machine_status.go

```diff
@@ -142,6 +143,7 @@ func NewClusterMachineStatusController() *ClusterMachineStatusController {
 					cmsVal.ApidAvailable = machine.TypedSpec().Value.Connected
 				}
 
+				// should we also mark it as not ready when the clustermachine is tearing down (?)
```

Comment:
as long as there's some help message in the UI that explains why it's not ready I guess it's fine